.github/workflows/ci.yml

```diff
@@ -28,6 +27,20 @@ jobs:
           script: |
             core.setFailed('Please push a commit to trigger the build. To push an empty commit you can use `git commit --allow-empty -m "Trigger Build"`') 
         if: ${{ github.run_attempt > 1 }}
+  validate_e2e_execution:
+    name: Validate pull_request files
+    runs-on: ubuntu-latest
+    outputs: 
+      SHOULD_RUN_E2E: ${{ steps.save-e2e-output-result.outputs.SHOULD_RUN_E2E }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Determine whether to run e2es
+        id: save-e2e-output-result
+        run: |         
+          source ci-scripts/validate-e2e-execution.sh
+          echo "::set-output name=SHOULD_RUN_E2E::$(echo $RUN_E2E)"
```

Comment:
i dont see this used in an if statement to determine whether we want to run tests or not.
i see it on your testing branch. im guessing this is because this PR is most likely still in-progress?

Reply:
nay it is here because that's how to set outputs in github actions